### PR TITLE
Update/v24.0.1

### PR DESCRIPTION
**Pull request by kn0wmad**

### CODE REVIEW

**Review by Blu-J — APPROVED** [review]

File: migrations/eq_23_0_0_lt_24_0_0.sh [hunk 1]
```diff
@@ -5,7 +5,6 @@ set -ea
 if [ $1 = "from" ]; then 
     yq -i '.advanced.peers.addnode |= map(select(.hostname != ~ or . == "*"))' /root/.bitcoin/start9/config.yaml
     yq -i '.advanced.peers.addnode.[] |= {"hostname":., "port":~}' /root/.bitcoin/start9/config.yaml
-    yq -i '(.advanced.mempool.mempoolfullrbf) = false' /root/.bitcoin/start9/config.yaml
```
Comment (Blu-J) [inline migrations/eq_23_0_0_lt_24_0_0.sh:8]: Why remove?

File: manifest.yaml [hunk 2]
```diff
@@ -235,7 +235,7 @@ migrations:
       mounts:
         main: /root/.bitcoin
       inject: false
-    "=24.0.0":
+    "=24.0.0 <25.0.0":
```
Comment (chrisguida) [inline manifest.yaml:238]: Can you explain this change?

**Review by Blu-J — APPROVED** [review]
Code looks ok to me, not tested by me

File: migrations/eq_23_0_0_lt_24_0_0.sh (hunk 1 above)
Comment (chrisguida) [inline migrations/eq_23_0_0_lt_24_0_0.sh:8]: Also this?

**Review by chrisguida — COMMENTED** [review]
See comments

File: manifest.yaml (hunk 2 above)
Comment (kn0wmad) [reply manifest.yaml:238]: 24.0.1 is > 24.0.0

File: migrations/eq_23_0_0_lt_24_0_0.sh (hunk 1 above)
Comment (kn0wmad) [reply migrations/eq_23_0_0_lt_24_0_0.sh:8]: Coming from those versions there should not exists a fullrbf config option

Comment (kn0wmad) [reply migrations/eq_23_0_0_lt_24_0_0.sh:8]: See above reply to blu-j

File: manifest.yaml (hunk 2 above)
Comment (chrisguida) [reply manifest.yaml:238]: Are you sure this is valid syntax here? The = I thought is only valid for a single version.

This says "equals 24.0.0 or is less than 25.0.0"

That doesn't make sense to me, how is that interpreted @dr-bonez ?

File: migrations/eq_23_0_0_lt_24_0_0.sh (hunk 1 above)
Comment (chrisguida) [reply migrations/eq_23_0_0_lt_24_0_0.sh:8]: Right, but then why was the line here before?

**Review by dr-bonez — APPROVED** [review]

**Pull request merged**